### PR TITLE
Add node v6 support. Fixes #106

### LINKED ISSUE #106: Add support for Node.js 6
When I run tests `npm test`, I get this error:

```
$ npm test

> mock-fs@3.8.0 pretest /home/ppitonak/workspace/mock-fs
> eslint benchmarks lib test

> mock-fs@3.8.0 test /home/ppitonak/workspace/mock-fs
> mocha --recursive test

/home/ppitonak/workspace/mock-fs/lib/index.js:36
  throw new Error('Unsupported Node version: ' + nodeVersion);
  ^

Error: Unsupported Node version: 6.0.0
    at Object.<anonymous> (/home/ppitonak/workspace/mock-fs/lib/index.js:36:9)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Module.require (module.js:466:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/ppitonak/workspace/mock-fs/test/integration/filecount.spec.js:4:12)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Module.require (module.js:466:17)
    at require (internal/module.js:20:19)
    at /home/ppitonak/workspace/mock-fs/node_modules/mocha/lib/mocha.js:219:27
    at Array.forEach (native)
    at Mocha.loadFiles (/home/ppitonak/workspace/mock-fs/node_modules/mocha/lib/mocha.js:216:14)
    at Mocha.run (/home/ppitonak/workspace/mock-fs/node_modules/mocha/lib/mocha.js:468:10)
    at Object.<anonymous> (/home/ppitonak/workspace/mock-fs/node_modules/mocha/bin/_mocha:403:18)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:159:18)
    at node.js:444:3
npm ERR! Test failed.  See above for more details.
```

### PR DESCRIPTION
**Pull request by tmcw**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ var versions = {
   '2.x.x': 'fs-2.0.0.js',
   '3.x.x': 'fs-3.0.0.js',
   '4.x.x': 'fs-4.0.0.js',
-  '5.x.x': 'fs-5.0.0.js'
+  '5.x.x': 'fs-5.0.0.js',
+  '6.x.x': 'fs-5.0.0.js'
```
Comment (jprichardson) [inline lib/index.js]: The file you committed is `fs-6.0.0`, but here you have `fs-5.0.0.js`. Presumably a typo?

**Comment by jkuri** [comment]
@tschaub can you make a release on npm please after this will be in?

**Comment by TheAlphaNerd** [comment]
:+1: thanks for getting this together... just noticed this was breaking in our CI.

Would you be open to adding support for v7.0.0-pre? This would help with some of the smoke testing efforts, specifically to smoke test packages who use mock-fs against master

**Comment by tschaub** [comment]
Released as mock-fs@3.9.0.  Thanks @tmcw.

@TheAlphaNerd I'd welcome any support on v7.0.0-pre.

**Pull request merged**